nesta/packages/dq/general.py

```diff
@@ -0,0 +1,132 @@
+import pandas as pd
+import numpy as np
+import itertools
+
+def missing_values(input):
+    """missing_values
+
+    Args:
+        input (:obj:`iter` of :obj:`obj`): A sequence of objects.
+
+    Returns:
+        missing_count_df (:obj:`pandas.core.series.Series`): A series with column names (index) and missing value frequency (column)
+
+
+    """
+    df = pd.DataFrame(input)
+    missing_count_df = df.isnull().sum().sort_values(ascending=False)
+    return missing_count_df
+
+def missing_value_percentage_column_count(input):
+    """missing_value_percentage_column_count
+
+    Args:
+        input (:obj:`dict`): A dictionary of
+
+    Returns:
+        out_counts (:obj:`pandas.core.series.Series`): A series with mising value percentage ranges (index) and column frequency (column)
+
+    """
+
+    missing_bins = [i for i in range(100+1) if i%10 == 0]

```

Comment:
Maybe the user should be allowed to specify the bins as a kwarg, with a reasonable default?

Reply:
Add an arg for bins with default as `None` and do
```suggestion
if bins is None:
    bins = range(0, 101, 10)
```